ui/apps/platform/src/Components/CompoundSearchFilter/components/ConditionNumber.tsx

```diff
@@ -0,0 +1,149 @@
+import React, { useEffect, useState } from 'react';
+import { Button, NumberInput, SelectOption } from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+
+import SimpleSelect from './SimpleSelect';
+
+export type ConditionNumberProps = {
+    value: string;
+    onSearch: (value: string) => void;
+};
+
+const conditionMap = {
+    'Is greater than': '>',
+    'Is greather than or equal to': '>=',
+    'Is equal to': '=',
+    'Is less than or equal to': '<=',
+    'Is less than': '<',
+};
```

Comment:
I think both of these are valid additions, with the min/max/increment values being much more important to customize than the precision. Even though we don't have a use case other than CVSS right now, which is always 0-10, if the addition of these options is a quick follow up I think it makes sense to do it sooner to avoid context switching in the future.